Sources/HarmonizeSemantics/GrammarComponents/Provider/StatementsProviding.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  StatementsProviding.swift
+//  Harmonize
+//
+//  Created by Matheus Luz on 10/21/24.
+//
```

Comment:
We probably need to add PSS here

Reply:
Yeah, to keep a pattern we can replace `Created by` with PSS copyright